### PR TITLE
use custom Monaco to avoid tree shaking unused code from VS Code

### PR DESCRIPTION
**Pull request by akosyakov**
TODO:
 - [x] release new Monaco and update

### CODE REVIEW

File: packages/plugin-ext/src/typings/monaco/index.d.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-/// <reference types='monaco-editor-core/monaco'/>
+/// <reference types='@typefox/monaco-editor-core/monaco'/>
```
Comment (AlexTugarev) [inline packages/plugin-ext/src/typings/monaco/index.d.ts]: maybe it's out of scope here, but wouldn't it make sense to reference `@theia/monaco` here?

Comment (akosyakov) [reply packages/plugin-ext/src/typings/monaco/index.d.ts]: it is out of scope, please file an issue

**Comment by AlexTugarev** [comment]
verified, that API e.g. `canComputeDirtyDiff` and `computeDirtyDiff` is available again.

@akosyakov, something else to test?

**Comment by akosyakov** [comment]
@AlexTugarev maybe to test a bit Java editor, navigation, rename, issues which we hit during the last migration, if it looks good, merging?

**Comment by AlexTugarev** [comment]
with Java, the Outline is always one edit behind. but I'm not sure if this is already the case with current master.

for ts/js it's not the case.

**Comment by akosyakov** [comment]
@AlexTugarev it was always like that

**Comment by AlexTugarev** [comment]
for Java: "navigation, rename, issues which we hit during the last migration" looks good.

**Comment by akosyakov** [comment]
@AlexTugarev approved?

**Pull request merged**